modules/programs/nh.nix

```
@@ -90,6 +99,21 @@ in
 
         Install.WantedBy = [ "timers.target" ];
       };
+
+    };
+
+    launchd.agents.nh-clean = lib.mkIf (cfg.clean.enable && pkgs.stdenv.isDarwin) {
+      enable = true;
+      config = {
+        ProgramArguments = [
+          "${lib.getExe cfg.package}"
+          "clean"
+          "user"
+        ] ++ lib.optional (cfg.clean.extraArgs != "") cfg.clean.extraArgs;
+
```

```suggestion
        StandardErrorPath = "${config.home.homeDirectory}/Library/Logs/nh/nh.err.log";
        StandardOutPath = "${config.home.homeDirectory}/Library/Logs/nh/nh.out.log";
```

Would be nice to see whats going on

---

Thank you for the review! 
I'm hesitant to add this. Logs stored in these files are not auto rotated or cleaned up. So I rather enabling this be an explicit act instead of it being enabled by default. To me enabling the garbage collection is more of a set it and forget it type of thing. I'd like to see the logs only if something isn't working right and even then I'm okay with not have the historical logs and enabling it then. 

Do you think this is a must have?

---

Nah, that's fair. I can add it later as an opt in behavior.